python/social_bot/tasks.py

```diff
@@ -428,7 +433,7 @@ def run(self):
             agent_sentence = yield TeacherAction(reward=reward, done=done)
 
     @staticmethod
-    def get_icub_extra_obs(icub_agent):
+    def get_icub_extra_obs(agent_name, icub_agent):
```

Comment:
If this agent_name is just the string 'icub' for the icub_agent, then maybe use icub_agent.get_name() (adding that into pygazebo.cc?  If we allow two models of the same type icub into the world, then maybe add a get_model_type() into pygazebo.cc and call that to get agent_type?

Reply:
we'll have an Agent class latter. Use Agent instead.